src/components/TemplateLibrary/index.js

```diff
@@ -19,6 +19,9 @@ const LibraryComponent = () => {
   const [overallCounter, setOverallCounter] = useState({});
 
   useEffect(() => {
+    /**
+     * Loading the template library and storing them in the state.
```

Comment:
Mention the URL too. [https://templates.accordproject.org/](https://templates.accordproject.org/).